src/components/Todo/Todo.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Todo.css';
+import PropTypes from 'prop-types';
+
+export class Todo extends React.Component {
+  state = {
+    completed: this.props.completed,
+  }
+
+  handleChange = (event) => {
+    const { name, checked } = event.target;
+
+    this.setState({
+      [name]: checked,
```

Comment:
```suggestion
      completed: checked,
```

you have only one prop in state